src/TodoListItem.js

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+function TodoListItem(props){
+    return (<li key={props.item.id}>{props.item.title}</li>);
```

Comment:
The key gets added as an attribute to the component, not placed on the list item here.
<img width="300" alt="202211_0807AM-Brave Browser" src="https://user-images.githubusercontent.com/10133316/199716726-cbea67f8-9cd2-4eef-9072-51b911d29f9e.png">


It's a good idea to keep the browser's development tools open so that you see the errors/warnings that React prints out.